website/snippets/_adapters-trusted.md

```diff
@@ -127,17 +138,6 @@
   padding: 0rem 0.75rem; display:inline-block'>dbt Core</div>"
     icon="snowflake"/>
 
-<Card
-    title="Spark"
```

Comment:
updated this to use full trademarked name "Apache Spark" 

https://spark.apache.org/